templates/commands/render_test.go

```diff
@@ -167,7 +167,7 @@ func TestRealRun(t *testing.T) {
 	t.Parallel()
 
 	specContents := `
-apiVersion: 'abc'
+apiVersion: 'abcxyz.dev/cli/v1alpha1'
```

Comment:
This feels weird? I would expect `cli.abcxyz.dev/v1alpha1` or `dev.abcxyz.cli/v1alpha1`

Reply:
SGTM, updated.